src/components/organisms/WithInformationCard/WithInformationCard.scss

```
@@ -0,0 +1,10 @@
+.information-title {
+  font-size: 22px;
+  font-weight: bold;
+}
+.information-subtitle {
+  font-size: 18px;
+}
+.information-description {
+  font-size: 18px;
+}
```

Ideally this would follow our latest BEM styling patterns as described in https://github.com/sparkletown/sparkle/pull/1401#pullrequestreview-660507743

- http://getbem.com/naming/

```suggestion
.WithInformationCard {
  &__title {
    font-size: 22px;
    font-weight: bold;
  }
  
  &__subtitle {
    font-size: 18px;
  }

  &__description {
    font-size: 18px;
  }
}
```

---

Actually, why are these styles being set here in `WithInformationCard`, yet seemingly designed to override the styles in `src/components/molecules/InformationCard/InformationCard.scss`? Would they not be better set in `InformationCard.scss`?

Ideally the BEM pattern described above would be used directly in `InformationCard.scss`, though there would be couple of places that would need updating across the codebase to use those new names when you do:

- https://github.com/sparkletown/sparkle/search?q=information-card-container
- https://github.com/sparkletown/sparkle/search?q=information-card-text